x/ecocredit/simulation/genesis.go

```diff
@@ -225,32 +268,47 @@ func genGenesisState(ctx context.Context, r *rand.Rand, simState *module.Simulat
 
 	pKey2, err := ss.ProjectInfoTable().InsertReturningID(ctx,
 		&api.ProjectInfo{
-			Key:                 cKey2,
-			Id:                  "P02",
-			Admin:               accs[1].Address.Bytes(),
+			ClassKey:            cKey2,
+			Admin:               accs[1].Address,
 			ProjectJurisdiction: "AQ",
 			Metadata:            metadata,
+			Id:                  "C0201",
 		},
 	)
 	if err != nil {
 		return err
 	}
 
+	// project sequence
+	if err := ss.ProjectSequenceTable().Save(ctx, &api.ProjectSequence{
+		ClassKey:     cKey1,
+		NextSequence: pKey1 + 1,
```

Comment:
We should look up the sequence and use the next sequence here rather than `pKey +1`.

Reply:
updated